prow/getting_started_deploy.md

```diff
@@ -301,9 +301,9 @@ bazel run //prow/cmd/mkbuild-cluster -- \
 kubectl create secret generic build-cluster --from-file=cluster.yaml
 ```
 
-Mount this secret into the prow components that need it and set
-the `--build-cluster` flag to the location you mount it at. For instance, you
-will need to merge the following into the plank deployment:
+Mount this secret into the prow components that need it (at minimum: `plank`,
```

Comment:
This list seems like it will be misleading in the future as things change - don't the YAMLs call it out?

Reply:
I mean the `starter.yaml` that one would be deploying when following this getting started document.

Reply:
The `starter.yaml` assumes there is one cluster for both the Prow components and the jobs, so I do not really see it fit in there. Or should we add it commented-out?

Reply:
Oh, of course. Let's keep what you have here. I think it might be nice to remove all the default flag values we've got today so it's more explicit how everything threads together in the future.